cg/cli/workflow/microsalt/base.py

```diff
@@ -91,26 +84,22 @@ def config_case(
 ) -> None:
     """ Create a config file for a case or a sample analysis in microSALT """
 
-    microsalt_analysis_api: MicrosaltAnalysisAPI = context.obj["microsalt_analysis_api"]
-
-    case_id, sample_id = microsalt_analysis_api.resolve_case_sample_id(
+    analysis_api: MicrosaltAnalysisAPI = context.obj["analysis_api"]
+    case_id, sample_id = analysis_api.resolve_case_sample_id(
```

Comment:
I miss type hints 😢

Reply:
seems we cant type hint tuple unpacking.... :(